libs/genai/tests/integration_tests/test_chat_models.py

```diff
@@ -1,9 +1,14 @@
 """Test ChatGoogleGenerativeAI chat model."""
+
 import asyncio
 import json
 from typing import Generator, List, Optional, Type
 
 import pytest
+from google.generativeai.types import (  # type: ignore[import]
```

Comment:
why do we need this change?

Reply:
In my environment, this error is shown.
https://github.com/google-gemini/generative-ai-python/issues/264

Do you know or something hint about this?

Reply:
please, feel free to add a `# type: ignore[import]` hint but let's continue importing from langchain_google_genai, please

Reply:
@lkuligin 
from langchain_google_genai._enums import helped me.
It looks import google.ai.generativelanguage_v1beta works without ignore.